source/Related-Projects/Ogma.rst

```diff
@@ -0,0 +1,31 @@
+Ogma
+====
+
+NASA's `Ogma <https://github.com/nasa/ogma>`_ is an open source tool to generate runtime monitoring applications.
+
+The purpose of Runtime Verification is to monitor a system at runtime and detect anomalies or violations of expected properties. RV concerns itself with detecting the violations; it is the user's responsibility to determine how to address them.
+
+Ogma facilitates the generation of correct monitors by translating high-level languages into self-contained monitoring applications.
+
+Ogma supports a number of source languages, including `Copilot <https://github.com/copilot-language/copilot>`_, `FRET <FRET>` and Lustre. The applications can be generated for the Robot Operating System (SpaceROS) and `NASA's Core Flight System (cFS) <https://github.com/nasa/cFS/>`_, among others.
```

Comment:
Looks like the entry for FRET is missing a proper URL.

Reply:
My bad. I was under the impression that, with this format, it would pick up the page on FRET in the same directory. I'll double check the syntax.

Reply:
I believe this is now resolved.